docs/source/conf.py

```diff
@@ -81,7 +81,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = 'python'
```

Comment:
Maybe this was intended for `highlight_language`. However, the default value of `highlight_language` should be sufficient and, if not, `python3` would be a better choice that `python`.

Reply:
Yes, `python3` would be better. There is also `nbsphinx_kernel_name` worth noting

Reply:
Okay. Well, I think adding a `highligh_language` setting may be beyond the scope of the present PR, but is certainly unrelated to fixing the `language` setting.